test/unit/sthDatabase_test.js

```diff
@@ -1114,6 +1136,95 @@ function retrievalTests() {
     });
 }
 
+describe('return 500 test', function() {
+    this.timeout(5000);
+    describe('database connection', function() {
+        let contextResponseNumericWithFixedTimeInstantV1;
+
+        before(function(done) {
+            sthDatabase.connect(
+                DATABASE_CONNECTION_PARAMS,
+                function(err, connection) {
+                    expect(err).to.be(null);
+                    expect(connection).to.equal(sthDatabase.connection);
+                    // It is needed to start the server for e2e test using HTTP endpoint. Otherwise, this step is not needed
+                    sth.sthServer.startServer(sthConfig.STH_HOST, sthConfig.STH_PORT, function() {
+                        sthDatabase.closeConnection(function(err) {
+                            expect(err).to.be(null);
+                            expect(sthDatabase.connection).to.be(null);
+                            contextResponseNumericWithFixedTimeInstantV1 = require('./contextResponses/V1contextResponseNumericWithFixedTimeInstant');
+                            done();
+                        });
+                    });
+                }
+            );
+        });
```

Comment:
Probably it can be simplified to just:

```suggestion
        before(function(done) {
            sthDatabase.connect(
                DATABASE_CONNECTION_PARAMS,
                function(err, connection) {
                    // It is needed to start the server for e2e test using HTTP endpoint. Otherwise, this step is not needed
                    sth.sthServer.startServer(sthConfig.STH_HOST, sthConfig.STH_PORT, function() {
                        sthDatabase.closeConnection(done);
                    });
                }
            );
        });
```